ui/src/app/views/confirm-delete-modal/component.tsx

```diff
@@ -50,9 +49,7 @@ export class ConfirmDeleteModal
   }
 
   render() {
-    return <React.Fragment>
-      {this.props.deleting &&
-      <Modal className='confirmDeleteModal'>
+    return <Modal className='confirmDeleteModal'>
```

Comment:
nit: while you're cleaning this, do we need the class?

Reply:
it may be used in the test spec

Reply:
Can't find anywhere that uses it.